packaging/leapp-supplements.spec

```diff
@@ -0,0 +1,111 @@
+# Define Globals
+%global leapp_datadir %{_datadir}/leapp-repository
+%global repositorydir %{leapp_datadir}/repositories
+%global custom_repositorydir %{leapp_datadir}/custom-repositories
+%global supplementsdir %{repositorydir}/system_upgrade_supplements
+
+%if 0%{?rhel} == 7
+    %define rpm_name leapp-supplements-el7toel8
+%endif
+%if 0%{?rhel} == 8
+    %define rpm_name leapp-supplements-el8toel9
+%endif
```

Comment:
I am ok with that. But thinking about that, wouldn't be better something like `leapp-upgrade-elXtoelY-supplements`? It's longer, but it keeps doors opened for `leapp-convert-el7toel7-supplements`, etc. In case something like that will be introduced in future. But it's a nitpick.

Reply:
Good thinking. It would probably even be more synchronous with the already present naming of `leapp-upgrade-elXtoelY`

will change